smart-contracts/contracts-common/concordium-contracts-common/src/types.rs

```diff
@@ -618,15 +650,21 @@ pub struct AccountPublicKeys {
 
 pub(crate) type CredentialIndex = u8;
 
-#[derive(crate::Serialize, Debug, SchemaType)]
+#[derive(crate::Serialize, Debug, SchemaType, PartialEq, Eq)]
+#[cfg_attr(
+    feature = "derive-serde",
+    derive(serde::Deserialize, serde::Serialize),
+    serde(tag = "SignatureSchema", content = "Signature")
```

Comment:
```suggestion
    serde(tag = "signatureScheme", content = "signature")
```

It's not a schema and we usually use camelCase naming.